css/style.css

```diff
@@ -417,6 +430,104 @@ p {
   border: solid;
 }
 
+.contact-section {
+  background-color: #fff;
+  height: 800px;
+}
+
+.contact-background {
+  position: absolute;
+  height: 800px;
+  width: 100%;
+  border-radius: 70px 0 0 0;
+  background-color: #6070ff;
+}
+
+.contact-stack {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 24px;
+  height: 800px;
+  gap: 30px;
+}
+
+.contact-banner {
+  text-align: center;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.contact-input {
+  padding: 15px 16px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: calc(100% - 34px);
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  margin-bottom: 20px;
+  outline: none;
+}
+
+.contact-input:active {
+  outline: 1px solid #172b4d;
+}
+
+.submit-btn {
+  padding: 12px 16px;
+  width: 140px;
+  border-radius: 8px;
+  background: #fff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #6070ff;
+  border: none;
+}
```

Comment:
While you did implement styling for the submit button, they don't follow the template design and it is a project requirement they do. Kindly fix this.

<img width="910" alt="Screen Shot 2022-05-31 at 13 03 38" src="https://user-images.githubusercontent.com/82426682/171254760-70a42ff2-60fd-412d-813f-fbced78c7ba7.png">

<img width="820" alt="Screen Shot 2022-05-31 at 13 03 53" src="https://user-images.githubusercontent.com/82426682/171254816-8e425753-7ca1-43d9-94ad-c8aa91a87443.png">